src/cloudant/design_document.py

```diff
@@ -169,13 +140,6 @@ def save(self):
                     if isinstance(view, QueryIndexView):
```

Comment:
Same [question](https://github.com/cloudant/python-cloudant/pull/168/files#r62887515) for line [138](https://github.com/cloudant/python-cloudant/blob/cefd64747f5ea45ef9782f6776f3977edce817ca/src/cloudant/design_document.py#L138)